src/test/resources/db/migration/V001__Create_schema.sql

```diff
@@ -0,0 +1,10 @@
+-- *** Product catalog ***
+CREATE TABLE Product
+(
+  id BIGSERIAL,
+  modificationCounter INTEGER NOT NULL,
+  title VARCHAR (255) NOT NULL,
+  description VARCHAR (4000),
+  price DECIMAL (16,2) NOT NULL,
+  CONSTRAINT PK_Product PRIMARY KEY(id)
+);
```

Comment:
isn't this schema be created as part of src/main/resources already?

Reply:
Deleted it, so the file in src/main/resources is used now